R/helpers_ggbetween_t_subtitles.R

```diff
@@ -115,12 +116,11 @@ subtitle_t_parametric <- function(data,
 
   # effect size object
   effsize_df <-
-    effsize::cohen.d(
```

Comment:
Since we are no longer using `effsize` anywhere. Can you get rid of any imports from this package and delete this dependency from the `DESCRIPTION` file? That should give us an idea if all instances of `effsize` have been effectively covered.

Reply:
All set I think

Reply:
I still see `effsize` in the `DESCRIPTION` file though.